### ISSUE TITLE
Remove InitDB functionality, rely on metal-api / metalctl apply instead

### ISSUE THREAD

**Issue by majst01** [metal-stack/masterdata-api#61]
The current approach of creating initial tenant and project entities is weird and counter intuitive.

We should instead expose all CRUD api endpoints for tenant and project in metal-api and then we can create initial entities with metalctl apply -f.

Thoughts ?

**Comment by Gerrit91** [comment]
Yes, let's remove this mechanism if possible. metal-api should be able to handle this.

**Comment by majst01** [comment]
Required PR https://github.com/metal-stack/metal-api/pull/357

**Comment by Gerrit91** [comment]
Probably not the most relevant topic. We can consider this in the new tenant-api repo or silently ignore this implementation and just migrate to metalctl apply.
